client/verta/tests/test_entities.py

```diff
@@ -466,6 +466,19 @@ def test_clone_into_expt(self, client):
         assert old_run_msg.observations == new_run_msg.observations
         assert old_run_msg.artifacts == new_run_msg.artifacts
 
+    def test_log_attribute_overwrite(self, client):
+        experiment_run = client.set_experiment_run(attrs={"str-attr": "attr", "int-attr": 4, "float-attr": 0.5})
+
+        with pytest.raises(ValueError) as excinfo:
+            experiment_run.log_attribute("str-attr", "some-attr")
+
+        assert "already exists" in str(excinfo.value)
+
+        experiment_run.log_attribute("str-attr", "new-attr", True)
+        experiment_run.log_attributes({"int-attr": 5, "float-attr": 0.3, "bool-attr": False}, True)
+
+        assert experiment_run.get_attributes() == {"str-attr": "new-attr", "int-attr": 5, "float-attr": 0.3, "bool-attr": False}
+
```

Comment:
Highly encourage storing all these attributes as variables, so that the assert is easier to read and also to help mitigate typos.

Reply:
@nhatsmrt Ah, no, I means entire dictionaries, such as `{"str-attr": "attr", "int-attr": 4, "float-attr": 0.5}` should be a single variable.

Something like
```python
initial_attrs = {"str-attr": "attr", "int-attr": 4, "float-attr": 0.5}
new_attrs = {"str-attr": "new-attr", "int-attr": 5, "float-attr": 0.3, "bool-attr": False}
single_new_attr = new_attrs.popitem()

# ...

run.log_attribute(*single_new_attr, overwrite=True)  # or whatever works
run.log_attributes(new_attrs, overwrite=True)

expected_attrs = initial_attrs.copy()
expected_attrs.update([single_new_attr])  # or whatever works
expected_attrs.update(new_attrs)
assert run.get_attrs() == expected_attrs
```

Unless I'm missing a detail, this would reduce a lot of redundancy in this test and make it easier to read.

Reply:
Fixed.